<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>691</width>
    <height>418</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <widget class="QWidget" name="stackedWidgetPage1">
       <layout class="QVBoxLayout" name="verticalLayout"/>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>220</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>240</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tabWidgetPage1">
          <attribute name="title">
           <string>Manual</string>
          </attribute>
          <layout class="QFormLayout" name="formLayout">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
           </property>
           <property name="horizontalSpacing">
            <number>9</number>
           </property>
           <property name="verticalSpacing">
            <number>6</number>
           </property>
           <property name="margin">
            <number>9</number>
           </property>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Seleccionar cuadrante:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QToolButton" name="tbSelectCu">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Seleccionar área de Interés:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QToolButton" name="tbSelectZone">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QToolButton" name="tbApplyFilter">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QToolButton" name="tbDetectBorders">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="lbFiltroPA">
             <property name="text">
              <string>Aplicar filtro paso alto:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="lbBordes">
             <property name="text">
              <string>Detectar bordes:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QToolButton" name="tbSelectTWard">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Marcar Triángulo de Ward:</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Marcar Compresivo Principal:</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QToolButton" name="tbSelectCP">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QToolButton" name="tbDiagnosticate">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Determinar diagnóstico:</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabWidgetPage2">
          <attribute name="title">
           <string>Automático</string>
          </attribute>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="treeWidget">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="autoScroll">
          <bool>true</bool>
         </property>
         <property name="autoScrollMargin">
          <number>10</number>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="rootIsDecorated">
          <bool>true</bool>
         </property>
         <property name="itemsExpandable">
          <bool>true</bool>
         </property>
         <property name="animated">
          <bool>true</bool>
         </property>
         <attribute name="headerVisible">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Imagen</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Descripción</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Archivo">
    <property name="title">
     <string>&amp;Archivo</string>
    </property>
    <addaction name="actionAbrir_imagen"/>
    <addaction name="actionGuardar_resultados"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuFiltros">
    <property name="title">
     <string>Filtros</string>
    </property>
    <widget class="QMenu" name="menuDeteccion_de_bordes">
     <property name="title">
      <string>Deteccion de bordes</string>
     </property>
     <widget class="QMenu" name="menuDireccionales">
      <property name="title">
       <string>Direccionales</string>
      </property>
      <addaction name="actionEste"/>
      <addaction name="actionOeste"/>
      <addaction name="actionNorte"/>
      <addaction name="actionSur"/>
      <addaction name="actionSuroeste"/>
      <addaction name="actionSureste"/>
      <addaction name="actionNoroeste"/>
      <addaction name="actionNoreste"/>
      <addaction name="actionCompleto"/>
     </widget>
     <widget class="QMenu" name="menuBasadas_en_gradiente">
      <property name="title">
       <string>Basadas en gradiente</string>
      </property>
      <addaction name="separator"/>
      <addaction name="actionRoberts"/>
      <addaction name="actionSobel"/>
      <addaction name="actionPrewitt"/>
      <addaction name="actionFrei_Chen"/>
     </widget>
     <addaction name="menuDireccionales"/>
     <addaction name="menuBasadas_en_gradiente"/>
    </widget>
    <addaction name="actionPaso_Alto"/>
    <addaction name="actionPaso_Bajo"/>
    <addaction name="menuDeteccion_de_bordes"/>
   </widget>
   <addaction name="menu_Archivo"/>
   <addaction name="menuFiltros"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionAbrir_imagen">
   <property name="text">
    <string>Abrir imagen</string>
   </property>
   <property name="toolTip">
    <string>Abrir imagen para analizar.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionGuardar_resultados">
   <property name="text">
    <string>Guardar resultados</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionPaso_Alto">
   <property name="text">
    <string>Paso Alto</string>
   </property>
  </action>
  <action name="actionPaso_Bajo">
   <property name="text">
    <string>Paso Bajo</string>
   </property>
  </action>
  <action name="actionEste">
   <property name="text">
    <string>Este</string>
   </property>
  </action>
  <action name="actionOeste">
   <property name="text">
    <string>Oeste</string>
   </property>
  </action>
  <action name="actionNorte">
   <property name="text">
    <string>Norte</string>
   </property>
  </action>
  <action name="actionSur">
   <property name="text">
    <string>Sur</string>
   </property>
  </action>
  <action name="actionSuroeste">
   <property name="text">
    <string>Suroeste</string>
   </property>
  </action>
  <action name="actionSureste">
   <property name="text">
    <string>Sureste</string>
   </property>
  </action>
  <action name="actionNoroeste">
   <property name="text">
    <string>Noroeste</string>
   </property>
  </action>
  <action name="actionNoreste">
   <property name="text">
    <string>Noreste</string>
   </property>
  </action>
  <action name="actionRoberts">
   <property name="text">
    <string>Roberts</string>
   </property>
  </action>
  <action name="actionSobel">
   <property name="text">
    <string>Sobel</string>
   </property>
  </action>
  <action name="actionPrewitt">
   <property name="text">
    <string>Prewitt</string>
   </property>
  </action>
  <action name="actionFrei_Chen">
   <property name="text">
    <string>Frei-Chen</string>
   </property>
  </action>
  <action name="actionCompleto">
   <property name="text">
    <string>Completo</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tbSelectCu</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
